install_nuclear-boy.sh

```diff
@@ -0,0 +1,351 @@
+                  tables \
+                  future"
+
+setup_dependencies ()
+{
+  echo "--------------------------"
+  echo "Installing dependencies..."
+  echo "--------------------------"
+  # Check if the OS supports apt-get
+  if ! command -v apt-get &> /dev/null; then
+    echo "Unfortunately, your operating system does not support apt-get."
+    exit 1
+  fi
+  sudo apt-get -y update
+  sudo apt-get -y install ${apt_package_list}
+  echo "Dependencies installed"
+  sleep 2
+}
+
+setup_python_env () {
+  echo "---------------------------------"
+  echo "Setting up virtual environment..."
+  echo "---------------------------------"
+  if [ -d "${env_dir}" ]; then
+    echo "Virtual environment already exists. Deleting..."
+    rm -rf "${env_dir}"
+  fi
+  echo "Creating Python virtual env in ${env_dir}"
+  /usr/bin/python3 -m venv $env_dir
+  source $env_dir/bin/activate
+  pip3 install wheel
+  pip3 install ${pip_package_list}
+  echo "Python virtual env created."
+}
+
+set_ld_library_path(){
+  # hdf5 std directory
+  hdf5_libdir=/usr/lib/x86_64-linux-gnu/hdf5/serial
+  # need to put libhdf5.so on LD_LIBRARY_PATH
+
+  if [ -z $LD_LIBRARY_PATH ]; then
+    export LD_LIBRARY_PATH="${hdf5_libdir}:${env_dir}/lib"
+  else
+    export LD_LIBRARY_PATH="${hdf5_libdir}:${env_dir}/lib:$LD_LIBRARY_PATH"
+  fi
+}
+
+install_moab(){
+  echo "------------------"
+  echo "Installing MOAB..."
+  echo "------------------"
+  cd ${env_dir}
+  # clone and version
+  git clone --branch Version5.1.0 --single-branch https://bitbucket.org/fathomteam/moab moab-repo
+  cd moab-repo
+  mkdir -p build
+  cd build
+  # cmake, build and install
+  cmake ../ -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_libdir} \
+            -DBUILD_SHARED_LIBS=ON \
+            -DENABLE_PYMOAB=ON \
+            -DENABLE_BLASLAPACK=OFF \
+            -DENABLE_FORTRAN=OFF \
+            -DCMAKE_INSTALL_PREFIX=${env_dir}
+  make
+  make install
+  cd ${env_dir}
+  rm -rf "${env_dir}/moab-repo"
+  echo "MOAB installed"
+}
+
+install_dagmc(){
+  echo "-------------------"
+  echo "Installing DAGMC..."
+  echo "-------------------"
+  # pre-setup check that the directory we need are in place
+  cd ${env_dir}
+  # clone and version
+  git clone https://github.com/svalinn/DAGMC.git dagmc-repo
+  cd dagmc-repo
+  git checkout develop
+  mkdir build
+  cd build
+  # cmake, build and install
+  cmake ../ -DMOAB_CMAKE_CONFIG=$env_dir/lib/cmake/MOAB \
+            -DMOAB_DIR=$env_dir \
+            -DBUILD_STATIC_LIBS=OFF \
+            -DCMAKE_INSTALL_PREFIX=$env_dir
+  make
+  make install
+  cd ${env_dir}
+  rm -rf "${env_dir}/dagmc-repo"
+  echo "DAGMC installed"
+}
+
+install_openmc(){
+  echo "--------------------"
+  echo "Installing OpenMC..."
+  echo "--------------------"
+  cd ${env_dir}
+  git clone https://github.com/openmc-dev/openmc.git openmc-repo
+  cd openmc-repo
+  git checkout develop
+  mkdir bld
+  cd bld
+  # cmake, build and install
+  cmake .. -DCMAKE_INSTALL_PREFIX=${env_dir}
```

Comment:
I just skimmed through this PR and noticed this compile command for openmc doesn't appear to make use of DAGMC

Reply:
Thanks, I have also noticed that and updated. The `DAGMC` was not set.